Source/SvgElementStyle.cs

```diff
@@ -238,6 +238,16 @@ public virtual SvgFontWeight FontWeight
             set { Attributes["font-weight"] = value; IsPathDirty = true; }
         }
 
+        /// <summary>
+        /// Indicates the desired amount of condensing or expansion in the glyphs used to render the text.
+        /// </summary>
+        [SvgAttribute("font-stretch")]
+        public virtual SvgFontStretch FontStretch
+        {
+            get { return GetAttribute("font-stretch", true, SvgFontStretch.Inherit); }
```

Comment:
`font-stretch` initial value is `normal`.
So, 3rd argument is `SvgFontStretch.Normal`.

Reply:
Isn't the `SvgFontStretchConverter` handling default case?
```
public SvgFontStretchConverter() : base(SvgFontStretch.Normal) { }
```

Reply:
default value of property, not  converter.

Reply:
Some users seem to build SVG programmatically.

Reply:
Agree.